docs/docsite/rst/mazer/configure.rst

```diff
@@ -22,14 +22,8 @@ an example configuration file:
     server:
       ignore_certs: false
       url: https://galaxy-qa.ansible.com
-    content_path: ~/.ansible/content
-    options:
-      local_tmp: ~/.ansible/tmp
-      role_skeleton_ignore:
-         - ^.git$
-         - ^.*/.git_keep$
-      role_skeleton_path: null
-      verbosity: 0
+    collections_path: ~/.ansible/collections
+    global_collections_path: /usr/share/ansible/collections
```

Comment:
What about the API key?

Reply:
Added api_key to the examples and some help blurbs based on mazer readme